src/app/legacy/psammead/psammead-brand/src/index.jsx

```diff
@@ -165,7 +166,30 @@ const StyledBrand = ({ linkId, product, serviceLocalisedName = null, svg }) => (
   </>
 );
 
+const StyledDiv = styled.div`
+  color: ${props => props.theme.palette.WHITE};
+`;
+
+const PreviewEnvironmentIndicator = () => {
+  const { requestServiceChain } = useContext(RequestContext);
+
+  if (requestServiceChain && requestServiceChain.includes('MOZART')) {
+    return <StyledDiv aria-hidden>⚠️ Mozart</StyledDiv>;
+  }
+  return null;
+};
+
 const Brand = forwardRef((props, ref) => {
+  const [isPreview, setIsPreview] = useState(false);
+
+  useEffect(() => {
+    const hostnameMatchesPreview = window.location.hostname.match(/preview/g);
+    const isPreviewEnvironment =
+      hostnameMatchesPreview && hostnameMatchesPreview.length > 0;
+
+    setIsPreview(isPreviewEnvironment);
+  }, []);
+
```

Comment:
Just need to double check that no CLS or LCP issues happen or any weird behaviour with child components, as using state might cause a re-render. 

Not actually sure if react does a re-render when you set an already false state value to false, hopefully it's clever enough, or you could be doubly safe and just set the state when isPreviewEnvironment is true

Reply:
Gosh I never thought about LCP/CLS 😬 Luckily this is only on the preview environment, but all the same, it's best to check. I'll ensure not to set state unless preview env is true & see if that causes a re-render or not.

Reply:
But of course this code will run regardless of which environment it's on. I'll do a quick comparison between the test environment & preview with this banner to see if there's a difference in Web Vital scores.

Reply:
It should be fine, as I don't think there's anything that will actually make react change anything in the DOM, but best to double check